### PR TITLE
Update ansible-runner for best practices

### PR DESCRIPTION
**Pull request by mlangbehn**
Signed-off-by: Matt Langbehn <matthew.langbehn@gmail.com>

### CODE REVIEW

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.org/check_github/BonnyCI/hoist/300/1491414355.39/bonnyci-run-check-multinode : POST_FAILURE in 16s

**Comment by mlangbehn** [comment]
recheck

**Comment by anne-bonny** [comment]
Build failed.

- http://logs.bonnyci.org/check_github/BonnyCI/hoist/300/1491416306.84/bonnyci-run-check-multinode : FAILURE in 1m 05s

**Comment by omgjlk** [comment]
Failing tests with inability to find a playbook. Something in the quoting maybe is going wrong.

**Comment by mlangbehn** [comment]
jlk: Yeah, I saw that in the log, so I tested ansible-runner in my local dev environment and it succeeded. I'm working on testing test/test-ansible-runner.sh to see if that's where the issue is.

File: roles/ansible-runner/files/usr/local/bin/ansible-runner [hunk 1]
```diff
@@ -18,44 +17,44 @@ if ! ([ -n "$ANSIBLE_PLAYBOOK" ] && [ -n "$ANSIBLE_INVENTORY" ]); then
...
 fi
 
-args="$@"
+args="$*"
 if [ -n "$ssh_user" ]; then
     args="$args -e ansible_ssh_user=$ssh_user"
 fi
-$ansible_venv/bin/ansible-playbook -vv -i $ANSIBLE_INVENTORY $args $ANSIBLE_PLAYBOOK 2>&1 | tee -a $logfile
-date 2>&1 | tee -a $logfile
+"$ansible_venv"/bin/ansible-playbook -vv -i "$ANSIBLE_INVENTORY" "$args" "$ANSIBLE_PLAYBOOK" 2>&1 | tee -a "$logfile"
```
Comment (gandelman-a) [inline roles/ansible-runner/files/usr/local/bin/ansible-runner]: The issue is the quoting of $args here.  With quotes, it'll pass along an empty string as an argument if args is empty, as opposed to passing along no argument if its unquoted.

Comment (mlangbehn) [reply roles/ansible-runner/files/usr/local/bin/ansible-runner]: Thanks.

**Review by gandelman-a — APPROVED** [review]

**Pull request merged**